src/Adapter/SsiCache.php

```diff
@@ -44,14 +45,13 @@ class SsiCache implements CacheAdapterInterface
 
     /**
      * NEXT_MAJOR: make $argumentResolver mandatory when dropping sf < 3.1.
```

Comment:
Probably this should be addressed too

Reply:
we are not dropping support for sf 3.1 yet, so this will probably stay for next major?

Reply:
Maybe we should drop up to 3.4 on the majors to start clean. I already tried to introduce this on a previous discussion n dev-kit, still not sure what we want to do as a team.. IMO having to maintain 3 versions of Symfony on new major releases is not a great idea

Reply:
I agree, do you have a link to that issue and we can continue there? I would be happy to drop < 3.4

Reply:
here: https://github.com/sonata-project/dev-kit/pull/389 .

it was not an issue tho. On dashboard bundle is already done, so maybe it can be done here too

Reply:
I think we should create an issue where we can do a vote or something because only you and core talked there and everyone should be included in this decision.

Reply:
Tomorrow I will create it, unless you create it first :P

Reply:
I leave it to you. :smile: